### DISCUSSION TITLE
What is "Average key norm" and why does it keep going up?

### DISCUSSION THREAD

**Discussion by demoran23** [bmaltais/kohya_ss#1478] [Q&A]
What is "Average key norm" and why does it keep going up?

**Comment by DarksealStudios** [comment]
i would love to know as well.

**Comment by TeKett** [comment]
Whould love to know as well

**Comment by zacvine** [comment]
it's like dark matter...the greatest scientists all agree that it's there, that it's real and that it's of great importance....but none of them really know quite what it is...
